sel4test-sim/build.py

```diff
@@ -18,13 +18,13 @@
 def run_simulation(manifest_dir: str, build: Build):
     """Run one simulation build and test."""
 
-    expect = f"\"{build.success}\""
+    expect = '{ "%s" {exit 0} timeout {exit 1} }' % build.success
 
     script = [
         ["../init-build.sh"] + build.settings_args(),
         ["ninja"],
         ["bash", "-c",
-         f"expect -c 'spawn ./simulate; set timeout 3000; expect {expect}' | tee {junit_results}"]
+         f"expect -c 'spawn ./simulate; set timeout 1200; expect {expect}' | tee {junit_results}"]
```

Comment:
Nitpicking: could re-use things here also?
```suggestion
         f"{' '.join(sim_script(build_success))} | tee {junit_results}"
```

Reply:
Also just joining the parts of the list with space is not guaranteed to result in a valid command line. You'd need to quote each part separate and make sure that the quote character used is not used on the inside of the argument. I wouldn't go there.

Reply:
An no, there is another quote layer in there. Maybe a long term improvement could be using https://pexpect.readthedocs.io/en/stable/install.html